File: gyroTest/src/main/java/com/stulnikov/arduino/arduinoBt/gyrotest/gyrotest/MainView.java
package com.stulnikov.arduino.arduinoBt.gyrotest.gyrotest;

/**
 * @author alexeystulnikov 12/25/16.
 */

public interface MainView extends BaseView {
    void toggleRobotMode(boolean robotMode);

    void showAngle(float angle);

    void showX(float x);

    void showY(float y);

    void showZ(float z);

    void showReceivedData(String data);

    void onDeviceConnected();

    void onDeviceDisconnected();
}

File: gyroTest/src/main/kotlin/com/stulnikov/arduino/arduinoBt/gyrotest/gyrotest/MainView.kt
package com.stulnikov.arduino.arduinoBt.gyrotest.gyrotest

/**
 * @author alexeystulnikov 12/25/16.
 */
interface MainView : BaseView {
    fun toggleRobotMode(robotMode: Boolean)
    fun showAngle(angle: Float)
    fun showX(x: Float)
    fun showY(y: Float)
    fun showZ(z: Float)
    fun showReceivedData(data: String)
    fun onDeviceConnected()
    fun onDeviceDisconnected()
    fun showEnableBluetooth()
}